packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/cards/ChartsCard.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from 'react';
+import { DateTime, Interval } from 'luxon';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { MigrationModelGroupVersionKind, V1beta1Migration } from '@kubev2v/types';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartTooltip } from '@patternfly/react-charts';
+import {
+  Card,
+  CardActions,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Dropdown,
+  DropdownItem,
+  Flex,
+  KebabToggle,
+} from '@patternfly/react-core';
+import chart_color_blue_200 from '@patternfly/react-tokens/dist/esm/chart_color_blue_200';
+import chart_color_green_400 from '@patternfly/react-tokens/dist/esm/chart_color_green_400';
+import chart_color_red_100 from '@patternfly/react-tokens/dist/esm/chart_color_red_100';
+
+import { MigrationsCardProps } from './MigrationsCard';
+
+interface MigrationDataPoint {
+  dateLabel: string;
+  value: number;
+}
+
+const toStarted = (m: V1beta1Migration): string => m.status.started;
+const toFinished = (m: V1beta1Migration): string => m.status.completed;
+const hasTimestamp = (timestamp: string) => timestamp && DateTime.fromISO(timestamp).isValid;
+const toDateTime = (timestamp: string): DateTime => DateTime.fromISO(timestamp);
+const isLast7Days = (date: DateTime) => date.diffNow('days').get('days') <= 7;
+const isLast24H = (date: DateTime) => date.diffNow('hours').get('hours') <= 24;
+const toDayLabel = (date: DateTime): string => date.toFormat('LLL dd');
+const toHourLabel = (date: DateTime): string => date.toFormat('HH:mm');
+const createTimeBuckets = (isDaysViewSelected: boolean) =>
```

Comment:
this should get a timegap in seconds and not be restricted  to days or hours

Reply:
It's a good practice to leave time calculations to the library. Different time units require different math - check https://moment.github.io/luxon/#/math

Reply:
I'm missing something, seconds are seconds in all time systems and units.

using  base units (seconds) is the best and safest way to reduce a list into buckets (in this specific case using the base unit seconds is safer then using a library), please use seconds for this calculation.